benches/packet_processing.rs

```diff
@@ -22,11 +22,9 @@ fn process_packet_before_send(
 }
 
 fn send_unreliable_benchmark(c: &mut Criterion) {
-    let config = NetworkConfig::default();
-    let mut connection = VirtualConnection::new(
-        SERVER_ADDR.parse().unwrap(),
-        Arc::new(NetworkConfig::default()),
-    );
+    let config = Config::default();
+    let mut connection =
+        VirtualConnection::new(SERVER_ADDR.parse().unwrap(), Arc::new(Config::default()));
```

Comment:
Unwrap alert! Unwrap alert!

Can we use expect?

Reply:
is this code even run? Yeah, look like it's not run at all because it would be completely broken.